src/main/java/com/commercetools/util/IOUtil.java

```diff
@@ -0,0 +1,29 @@
+package com.commercetools.util;
+
+import javax.annotation.Nonnull;
+import javax.servlet.ServletInputStream;
+import javax.servlet.http.HttpServletRequest;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.util.stream.Collectors;
+
+/**
+ * Util for working with input/output operations
+ */
+public class IOUtil {
```

Comment:
**`final`**

also, it is recommended to keep the private constructor at the bottom, cos it is empty and useless, so when you open the class sources you see immediately the significant methods

Reply:
https://github.com/commercetools/commercetools-paypal-plus-integration/pull/78/commits/7d0e9c840115e0e640fc1be6d0450e8404090712